PWGHF/DataModel/HFSecondaryVertex.h

```diff
@@ -800,9 +774,36 @@ DECLARE_SOA_COLUMN(OriginMCGen, originMCGen, int8_t);               // particle
 DECLARE_SOA_COLUMN(FlagMCDecayChanRec, flagMCDecayChanRec, int8_t); // resonant decay channel flag, reconstruction level
 DECLARE_SOA_COLUMN(FlagMCDecayChanGen, flagMCDecayChanGen, int8_t); // resonant decay channel flag, generator level
 // mapping of decay types
-enum DecayType { XToJpsiPiPi = 0 }; // move this to a dedicated cascade namespace in the future?
+enum DecayType { XToJpsiToEEPiPi = 0,
+                 XToJpsiToMuMuPiPi }; // move this to a dedicated cascade namespace in the future?
 } // namespace hf_cand_x
 
+// X → Jpsi π+ π-
+// TODO: add pdg code for X (9920443), temporarily hardcode mass here:
+float massX = 3.872; // replace this with: "RecoDecay::getMassPDG(9920443)" when pdg is added
+template <typename T>
+auto CtX(const T& candidate)
+{
+  return candidate.ct(massX);
```

Comment:
Get the mass of X and Jpsi from `RecoDecay`.

Reply:
For the X this cannot be done because the X is not in the database
[FATAL] Cannot find particle mass for PDG code 9920443

Reply:
https://github.com/AliceO2Group/O2Physics/pull/117

Reply:
Please fix this in the next PR together with the update of the selector.